adafruit_rsa/pkcs1.py

```diff
@@ -229,15 +259,17 @@ def decrypt(crypto, priv_key):
     return cleartext[sep_idx + 1 :]
 
 
-def sign_hash(hash_value, priv_key, hash_method):
+def sign_hash(
+    hash_value: Optional[bytes], priv_key: PrivateKey, hash_method: str
```

Comment:
`hash_method` is perhaps another good candidate for Literal[]

`Literal['MD5', 'SHA-1', 'SHA-224', SHA-256', 'SHA-384', 'SHA-512']`

Reply:
Ah yes, good catch!